examples/load_ui/load_ui_1.py

```diff
@@ -0,0 +1,42 @@
+import sys
+import os
+
+# Set preferred binding
+os.environ["QT_PREFERRED_BINDING"] = "PySide:PyQt4"
```

Comment:
Could you make this `os.pathsep.join(["PySide", "PyQt4"])` so that it also runs on Windows?

(I would also be up for discussing an alternative syntax for this, having now seen it in the real-world, e.g. `"PySide,PyQt4"`)